examples/amp-consent/cmp-vendors.amp.html

```diff
@@ -79,6 +79,31 @@
     }
 
     </style>
+
+    <!-- iubenda example CSS -->
+    <style amp-custom>
+      .popupOverlay {
+        position: fixed;
+        top: 0;
+        bottom: 0;
+        left: 0;
+        right: 0;
+      }
+    
+      amp-iframe {
+        margin: 0;
+      }
+      
+      amp-consent.amp-active {
+        top: 0;
+        bottom: 0;
+        left: 0;
+        right: 0;
```

Comment:
nit; remove this styling since we have them in your documentation.

Reply:
Done.